core/util/inAppLink.jsx

```diff
@@ -1,9 +1,40 @@
+function openUrl(url) {
+  SafariViewController.isAvailable(function (available) {
```

Comment:
@johnthepink we should probably wrap this with either an existence check on `SafariViewController` or a more general `try/catch`